### ISSUE TITLE
New Questions should go to the bottom of the list

### ISSUE THREAD

**Issue by ccoenen** [thm-projects/arsnova.click#12]
Right now in quiz manager, if I add a question, it gets added to the top of the list of questions. I find this very odd and I would propose to add the new question to the bottom of the list.

When I create a quiz, I usuall go about it in chronological order anyway. (I am working under the assumption that most people will probably do this.) So, for each question I create, I need to manually send it to the bottom of the stack of questions.

![grafik](https://user-images.githubusercontent.com/124909/81821491-30ed8f80-9532-11ea-9d4c-ff6a7feb84ad.png)

**Comment by trayhem** [comment]
We moved the inserted question to the top because of the changes to the view of questions in the quiz manager. If you have a lot of questions you would would maybe miss that a question has been added because you would need to scroll down.

An alternative would be to scroll automatically to the bottom but this would lead to a visual interrupt because the window will move down.

Maybe we could revert this with #11 (I think this is an very interesting idea) or we could offer two buttons in the "new question" button - one to add the question to the top and one to add it to the button.
